app/assets/stylesheets/modules/header.scss

```diff
@@ -0,0 +1,5 @@
+#header-navbar {
+  .navbar-brand {
+    height: 52px;
```

Comment:
How does this relate to `$navbar-height` below?

Reply:
`navbar-height` is set via bootstrap, but Blacklight hardcodes a height of `50px` which cuts off the bottom of this image. https://github.com/projectblacklight/blacklight/blob/master/app/assets/stylesheets/blacklight/_header.scss#L24

Reply:
I should add this is only for the `.navbar-brand` modification.